screeps-start.js

```diff
@@ -21,6 +21,16 @@ const loadPackage = (dir) =>
 const isDependency = (pkg, [name, version]) =>
   pkg.includes(name) || version.includes(pkg);
 
+const parseVersionSpec = (spec) => {
+  const atIdx = spec.lastIndexOf("@");
+  if (atIdx === -1) {
+    return [spec, "latest"];
```

Comment:
So this defaults to saying that we upgrade those that are not pinned, correct?

Reply:
Yep. If you want a pin, you put ` - screepsmod-something@0.5.0` in config.yml and it'll stay pinned at that version.